Python/Tests/TestData/Coverage/Statements/test.py

```diff
@@ -0,0 +1,22 @@
+x = 1
+x += 2
+assert type(x) == int
+del x
+pass
+exec('x = 1')
+from sys import winver
+global x
+import sys
+print(x)
+try:
+    x = 1
+except:
+    pass
+    
+try:
+    x = 1
+finally:
+    pass
+    
+with open('foo.txt', 'w+'):
```

Comment:
We can't use `foo` in our code without upsetting the censors. `fob` or `spam` are better choices.